packages/shared/src/server/queues.ts

```diff
@@ -65,7 +66,19 @@ export const BatchExportJobSchema = z.object({
 export const TraceUpsertEventSchema = z.object({
   projectId: z.string(),
   traceId: z.string(),
+  type: z.literal("trace"),
```

Comment:
This here will break production. Currently, we have events in the queue without the type. If we deploy a worker which requires a type, this will fail.
Proposal: i think we should move the trace and dataset upsert event through their own queue.

Reply:
uuups, good catch, consider it fixed 🙌